daml-assistant/daml-helper/src/DA/Daml/Helper/Run.hs

```diff
@@ -711,6 +751,28 @@ getProjectName = do
     requiredE "Project must have a name" $
         queryProjectConfigRequired ["name"] projectConfig
 
+getProjectParties :: IO [String]
+getProjectParties = do
+    projectConfig <- getProjectConfig
+    requiredE "Project must have parties listed" $
```

Comment:
The error message seems incorrect a bit off, because it might appear if the parties are there but are ill formed, and only the left side of the Either has enough information to disambiguate the issue. I think something like `"Failed to read list of parties from project config"` would be better.

Reply:
This message was modelled of the exiting message in the code:
```
getProjectName :: IO String
getProjectName = do
    projectConfig <- getProjectConfig
    requiredE "Project must have a name" $
        queryProjectConfigRequired ["name"] projectConfig
```

Reply:
I see. That message has the same problem. Please change it to `"Failed to read project name from project config"` or so.

Reply:
will fix both messages in next PR

Reply:
@associahedron We could be more DRY here. The config lookup combinators should automate these messages.